README.md

```diff
@@ -99,30 +109,34 @@ Ensure that `Node.js` and `MySQL` are installed on your machine.
     node server.js
     ```
 
-After completing these steps, you should have the Drawn2Shoe application up and running on your local machine.
+After completing these steps, you should have the Drawn2Shoe application up and running on your local machine. 🎉
+
+8). Make the required changes and add the changed file . 
+9). Open a pull request from your branch to the main repository.
+10). Provide a clear description of your changes and why they're valuable.
```

Comment:
These points are specific to contributors and should not be included in the `Installation` heading. Please remove.

Reply:
> These points are specific to contributors and should not be included in the `Installation` heading. Please remove.
Hey , you are asking just to put these points under another subheading ....or just remove these points ??

Reply:
Remove these points.

Reply:
Hey , @dvjsharma , changes are done ! Kindly check.